app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  validates :content, presence: true
+  belongs_to :job
+
+  def self.reverse_chronological
+    Comment.order(created_at: :desc)
```

Comment:
because we are in a class method - we don't need to specify `Comment` here..

```ruby
def self.reverse_chronological
  order(created_at: desc)
end

```